httpie/core.py

```diff
@@ -16,7 +16,6 @@
 from httpie.plugins.registry import plugin_manager
 from httpie.status import ExitStatus, http_status_to_exit_status
 
-
```

Comment:
When you clean up the history, make sure there's not irrelevant changes like some of these removed blank lines